src/core/algorithms/sfd/cords/cords.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <fstream>
+#include <string>
+#include <vector>
+
+#include "algorithm.h"
+#include "config/tabular_data/input_table_type.h"
+#include "cords_sample.h"
+#include "frequency_handler.h"
+#include "model/table/column_layout_typed_relation_data.h"
+#include "model/types/types.h"
+
+namespace algos {
+
+class Cords : public Algorithm {
```

Comment:
> Why we don't introduce new class `SFDAlgorithm`?

While I'm not aware of any other algorithm for SDF discovery, it may be a good idea to introduce this additional class.